### PR TITLE
docs(python): Add missing `Series` methods to API reference

### PR DESCRIPTION
**Pull request by henryharbeck**
Closes https://github.com/pola-rs/polars/issues/17786

Missing doc items identified by looking at any methods without links [here](https://docs.pola.rs/api/python/stable/reference/series/index.html#:~:text=2%0A%20%20%20%20%20%20%20%203%0A%5D-,Methods%3A,-abs). Have validated that all works as expected against a local docs build.

Also note that `Series` does not implement many of the operator methods offered on DataFrame.

### CODE REVIEW

File: py-polars/docs/source/reference/series/computation.rst [hunk 1]
```diff
@@ -19,6 +19,7 @@ Computation
     Series.cos
     Series.cosh
     Series.cot
+    Series.count
```
Comment (MarcoGorelli) [inline py-polars/docs/source/reference/series/computation.rst:22]: would `aggregation` be a better section, as that's where it is for `Expr`?

Comment (henryharbeck) [reply py-polars/docs/source/reference/series/computation.rst:22]: Thanks for the question and the quick review. Yes, I do think that's a better section - thanks for that.

It is in both `aggregation` and `functions` for `Expr`, so initially that was why I moved it into `computation`. Upon reflection `aggregation` is indeed much better suited. Will update now.

**Review by MarcoGorelli — APPROVED** [review]
Nice one, thanks @henryharbeck !

If you the wanted to de-dupe `Expr.count` (remove it from "functions"?) I think that'd be good too - is it the only one that's duplicated or are there others?

**Comment by MarcoGorelli** [comment]
> If you the wanted to de-dupe Expr.count (remove it from "functions"?) I think that'd be good too - is it the only one that's duplicated or are there others?

edit: i asked about this, and devs are ok with some being in multiple places, so no need to change that for expr

**Comment by henryharbeck** [comment]
Thanks again @MarcoGorelli! 😃 
Apologies to disappear after you replied to me 1 minute later 😅, was getting late and then had a busy weekend

**Pull request merged**